README.md

```diff
@@ -12,6 +22,10 @@ Wave Recovery Tool is licensed under the terms of the GNU General Public License
 
 Wave Recovery Tool is developed by David Pace &lt;dev@davidpace.de&gt;
 
+## Author GUI 
```

Comment:
I would propose to remove this heading and change the heading above to `Authors`